.changeset/violet-dingos-shake.md

```diff
@@ -0,0 +1,6 @@
+---
+"@biomejs/biome_wasm": patch
```

Comment:
We don't have package called like this 🤔 we only have `@biomejs/wasm-bunder`, etc.

Reply:
We have https://www.npmjs.com/package/@biomejs/wasm-bundler, https://www.npmjs.com/package/@biomejs/wasm-nodejs, and https://www.npmjs.com/package/@biomejs/wasm-web so i've fixed it to add all those